tests/Fixtures/Transformations/features/bootstrap/TransformationAnnotationsContext.php

```diff
@@ -79,6 +79,26 @@ public function createUserNamesFromTableInRussian($tableRow) {
         return $tableRow['логин'];
     }
 
+    /** @Transform column:user,other user */
+    public function convertUsernameToUserInColumn(string $name): User {
+        return new User($name);
+    }
+
+    /** @Transform column:логин */
+    public function convertUsernameInRussianToUserInColumn(string $name): User {
+        return new User($name);
+    }
+
+    /** @Transform column:username */
+    public function convertUsernameToUserWithUsernameHeading(string $name): User {
+        throw new Exception('This should not be called');
+    }
+
+    /** @Transform column:hex age */
+    public function convertHexAgeToAge(string $hexAge): int {
+        return hexdec($hexAge);
+    }
+
```

Comment:
It might be better to define these in the attributes feature / profile instead of the annotations one? Just means fewer things to move if/when we drop annotations, and since it's a new feature it's probably reasonable to test only with attributes?